lib/webauthn/authenticator_response.rb

```diff
@@ -4,6 +4,9 @@
 
 module WebAuthn
   class VerificationError < Error; end
+  # `UnspecifiedRpIdError` is a caller error rather than an error in the webauthn
+  # response. So we directly subclass `Error`.
+  class UnspecifiedRpIdError < Error; end
```

Comment:
This happens if the caller doesn't specify *either* of `expected_origin` or `rp_id`. I figured it didn't make sense to call this a `VerificationError` — perhaps it should be an `ArgumentError`?